modules/utility.js

```diff
@@ -108,29 +108,31 @@ function replyEmbedError(msg, error) {
 }
 
 function splitMessages(msg, withPrefix = false) {
-    let command, args;
-    if (withPrefix) {
-        const split = msg.content.split(/ +/);
-        command = split[0].toLowerCase();
-        args = split.slice(1);
-    } else {
+    const regex = new RegExp(process.env.PREFIX, 'gi');
```

Comment:
Terjadi error di sini.
![Screenshot_1264](https://user-images.githubusercontent.com/87353624/149659821-6d5e8b0f-fd6f-4ea9-960d-2e9eb028a1f6.png)

Bahkan ketika chat biasa pun (mengobrol biasa), dapat menyebabkan bot hard crash.
![Screenshot_1265](https://user-images.githubusercontent.com/87353624/149659852-4e90f44c-d588-414d-9b4e-b8c3c1a7d57d.png)

Saya sebenarnya sudah dapat mengatasi masalah pada #19.
Dan juga membuat bot lebih mudah didevelop.

![Screenshot_1266](https://user-images.githubusercontent.com/87353624/149659919-37ae23ea-cc16-4bf2-8294-1cb5b264f685.png)

Jika setuju menggunakan cara seperti gambar di atas, maka akan saya PR segera.

Reply:
Unable to reproduce
![image](https://user-images.githubusercontent.com/31276236/149660721-de80331f-418d-4771-a1a1-4341b4b254ab.png)

Reply:
Erronya dibuat karena backslash tidak diescape, jadi prefix nya dirubah selain blackslash bisa atau jika ingin menggunakan backslah maka prefixnya perlu dibuat begini `\\`